packages/flutter/lib/src/material/pickers/date_utils.dart

```diff
@@ -120,3 +127,31 @@ int getDaysInMonth(int year, int month) {
   const List<int> daysInMonth = <int>[31, -1, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31];
   return daysInMonth[month - 1];
 }
+
+/// Returns an appropriate string to describe the start of a date range.
```

Comment:
When you say "appropriate", you mean "locale-appropriate", right? Not just, like, without cuss words. :-)